docs/source/tutorials.rst

```diff
@@ -1,17 +1,22 @@
+.. _tutorials:
+
 =================
 GSTools Tutorials
 =================
 
 In the following you will find several Tutorials on how to use GSTools to
 explore its whole beauty and power.
 
+
 .. toctree::
-   :maxdepth: 1
+   :includehidden:
+   :maxdepth: 3
```

Comment:
Why ``:maxdepth: 3``? We think 1 is enough, as the chapters are very short in general.